helm/prometheus-operator-app/templates/prometheus-operator/admission-webhooks/job-patch/job-createSecret.yaml

```diff
@@ -31,16 +31,12 @@ spec:
       {{- end }}
       containers:
         - name: create
-          {{- if (.Values.Installation) }}
-          image: "{{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.prometheusOperator.admissionWebhooks.patch.image.name }}:{{ .Values.prometheusOperator.admissionWebhooks.patch.image.tag }}"
-          {{ else }}
-          image: "{{ .Values.image.registry }}/{{ .Values.prometheusOperator.admissionWebhooks.patch.image.name }}:{{ .Values.prometheusOperator.admissionWebhooks.patch.image.tag }}"
-          {{ end }}
+          image: {{ .Values.prometheusOperator.admissionWebhooks.patch.image.repository }}:{{ .Values.prometheusOperator.admissionWebhooks.patch.image.tag }}
```

Comment:
We need to keep that part about installations as this app is also installed on the control plane

Reply:
Also, we need it to support China